.github/workflows/pr-tests.yml

```diff
@@ -31,6 +31,9 @@ jobs:
         uses: jwalton/gh-find-current-pr@v1
         id: findPr
 
+      - name: Install pnpm
+        run: pnpm add -g pnpm@8.9.0
```

Comment:
We don't need to run this because that action is already installing it (`FuelLabs/github-actions/setups/node@master`).


I'd recommend to add a `pnpm-version: 8.15.7` below that `node-version`.

Check this PR https://github.com/FuelLabs/fuels-wallet/pull/1235